### PR TITLE
Add block explorer api sidebar

### PR DESCRIPTION
**Pull request by AlbionaHoti**
# What :computer: 
* Add Block Explorer API on the sidebar

# Why :hand:
* To have it's section under the Block Explorer Menu

# Evidence :camera:
Screenshot 2023-10-31 at 16 59 24
Screenshot 2023-10-31 at 16 59 46

# Notes :memo:
* Any notes/thoughts that the reviewers should know prior to reviewing the code?

### CODE REVIEW

File: docs/.vuepress/sidebar/en.ts [hunk 1]
```diff
@@ -249,7 +249,7 @@ export const enSidebar = sidebar({
     {
       text: "Block Explorer", // required
       link: "/tools/block-explorer", // optional, which should be a absolute path.
-      children: ["/tools/block-explorer/block-explorer-menu", "/tools/block-explorer/contract-verification"],
+      children: ["/tools/block-explorer/block-explorer-menu", "/tools/block-explorer/contract-verification", "/tools/block-explorer/block-explorer-api"],
```
Comment (dutterbutter) [inline docs/.vuepress/sidebar/en.ts:252]: If block-explorer-api is first on the page it should be first link?

Comment (AlbionaHoti) [reply docs/.vuepress/sidebar/en.ts:252]: The API is the last subtitle on the block Explorer [readme file](https://aqwzx-zksync-v2-docs--pr772-add-block-explorer-a-zpgo5wmh.web.app/docs/tools/block-explorer/#block-explorer-api-1), even though the preview URL shows the [deleted part](https://aqwzx-zksync-v2-docs--pr772-add-block-explorer-a-zpgo5wmh.web.app/docs/tools/block-explorer/#block-explorer-api).

@Romsters mentioned having the API listed as the last one on the readme of Block Explorer and having the item as the last one. We can move it up.

Comment (Romsters) [reply docs/.vuepress/sidebar/en.ts:252]: API should be the last one

File: firebase.json [hunk 2]
```diff
@@ -322,6 +322,11 @@
           "source": "/docs/api/tools{,/**}",
           "destination": "/docs/tools/",
           "type": "302"
+        },
+        {
+          "source": "/docs/dev/community-tutorials/the-graph-tutorial.html",
```
Comment (idea404) [inline firebase.json:327]: not sure this change relates to the BE API docs

File: firebase.json [hunk 3]
```diff
@@ -647,6 +652,11 @@
           "source": "/docs/api/tools{,/**}",
           "destination": "/docs/tools/",
           "type": "302"
+        },
+        {
```
Comment (idea404) [inline firebase.json:656]: same here: not sure this change relates to the BE API docs

File: firebase.json (hunk 2 above)
Comment (AlbionaHoti) [reply firebase.json]: Removed, must have showed up from earlier

**Review by Romsters — APPROVED** [review]

**Pull request merged**